etc/rules/0505-ciscat_rules.xml

```diff
@@ -0,0 +1,70 @@
+<!--
+  -  CIS-CAT scanner rules
+  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+-->
+
+<!-- ID: 87400 - 87500 -->
+
+<!-- Example JSON event
+{"type":"scan_result","scan_id":1670424437,"cis-data":{"rule_id":"6.2.20","rule_title":"Ensure shadow group is empty","group":"Additional Process Hardening","description":"The shadow group allows system programs which require access the ability to read the /etc/shadow file. No users should be assigned to the shadow group.","rationale":"Any users assigned to the shadow group would be granted read access to the /etc/shadow file. If attackers can gain read access to the /etc/shadow file, they can easily run a password cracking program against the hashed passwords to break them. Other security information that is stored in the /etc/shadow file (such as expiration) could also be useful to subvert additional user accounts.","remediation":"Remove all users from the shadow group, and change the primary group of any users with shadow as their primary group.","result":"pass"}}
+-->
+
+<!-- CIS-CAT rules -->
+<group name="ciscat,">
+
+  <rule id="87401" level="3">
+    <decoded_as>json</decoded_as>
+    <field name="type">\.+</field>
+    <field name="scan_id">\.+</field>
+    <description>CIS-CAT events.</description>
+  </rule>
+
+  <rule id="87402" level="3">
+      <if_sid>87401</if_sid>
+      <field name="type">^scan_start$</field>
+      <description>CIS-CAT: assessment information for scan $(scan_id)</description>
+  </rule>
+
+  <rule id="87403" level="5">
```

Comment:
The rule level for correct results (_pass_, _notchecked_, _notselected_) should be `0` and `1` for _informational_. Please make these rules behave like https://github.com/wazuh/wazuh/blob/master/etc/rules/0385-oscap_rules.xml.